### PR TITLE
mgmt, local generation for Compute

### PR DESCRIPTION
**Pull request by v-hongli1**
# Description

1. Regenerated code for [issue#4303](https://github.com/Azure/sdk-release-request/issues/4303).
2. Skip the following test cases in `Playback` for `Compute` modules.
   `VirtualMachineManagedDiskOperationsTests.canCreateVirtualMachineWithHibernationEnabledUsingOsDisk`
   `VirtualMachineOperationsTests.canHibernateVirtualMachine`
   `VirtualMachineScaleSetOperationsTests.canCreateVMSSWithPlan`
3. Skip the following test cases in `Playback` for `sample` and `resouremanager` modules.
    `ComputeSampleTests.testCreateVirtualMachineUsingSpecializedDiskFromSnapshot`
    `ComputeSampleTests.testManageManagedDisks`
    `ComputeSampleTests.testCloneVirtualMachineToNewRegion`
    `KubernetesEncryptionTests.canCreateClusterWithDiskEncryption`
4. Re-record test result for test case `VirtualMachineEncryptionTests.canCreateVirtualMachineWithDiskEncryptionSet`
5. Add ignore items in `revapi.json` 

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/resourcemanager/azure-resourcemanager-compute/src/test/java/com/azure/resourcemanager/compute/VirtualMachineManagedDiskOperationsTests.java [hunk 1]
```diff
@@ -603,6 +604,7 @@ public void canCreateVirtualMachineWithManagedDiskInManagedAvailabilitySet() {
     }
 
     @Test
+    @DoNotRecord(skipInPlayback = true)
```
Comment (weidongxu-microsoft) [inline sdk/resourcemanager/azure-resourcemanager-compute/src/test/java/com/azure/resourcemanager/compute/VirtualMachineManagedDiskOperationsTests.java:607]: Do not skip it. Please keep the test. Just you don't need to record it. Use script to change api-version of that recording from main.

File: eng/code-quality-reports/src/main/resources/revapi/revapi.json [hunk 2]
```diff
@@ -285,6 +285,23 @@
...
+          "code": "java.method.parameterTypeChanged",
+          "old": "parameter com.azure.resourcemanager.compute.models.GalleryTargetExtendedLocation com.azure.resourcemanager.compute.models.GalleryTargetExtendedLocation::withStorageAccountType(===com.azure.resourcemanager.compute.models.StorageAccountType===)",
+          "new": "parameter com.azure.resourcemanager.compute.models.GalleryTargetExtendedLocation com.azure.resourcemanager.compute.models.GalleryTargetExtendedLocation::withStorageAccountType(===com.azure.resourcemanager.compute.models.EdgeZoneStorageAccountType===)",
+          "justification": "ADD YOUR EXPLANATION FOR THE NECESSITY OF THIS CHANGE"
+        },
+        {
+          "code": "java.field.removed",
+          "old": "field com.azure.resourcemanager.compute.models.StorageAccountType.STANDARD_SSD_LRS",
+          "justification": "Fix field removed."
         }
```
Comment (weidongxu-microsoft) [inline eng/code-quality-reports/src/main/resources/revapi/revapi.json:305]: We need to know the reason from Compute RP, especially the removal of `STANDARD_SSD_LRS`.

For model rename, we may use `--rename-model` to rename it back.

**Review by weidongxu-microsoft — CHANGES_REQUESTED** [review]
Please close this one.

We need to know why CRP breaks, and how we can handle that, before adding anything into revapi suppresion.

**Comment by v-hongli1** [comment]
@weidongxu-microsoft 

> Please close this RP.
> 
> We need to know why CRP breaks, and how we can handle that, before adding anything into revapi suppresion.

So is the release job of issue#4303 suspended?

**Comment by weidongxu-microsoft** [comment]
> @weidongxu-microsoft
> 
> > Please close this RP.
> > We need to know why CRP breaks, and how we can handle that, before adding anything into revapi suppresion.
> 
> So is the release job of issue#4303 suspended?

No, it is not. We need to clarify with service.

**Comment by weidongxu-microsoft** [comment]
Simple rule: do not modify "revapi.json" until we've discussed it.

**Comment by weidongxu-microsoft** [comment]
Please schedule a meeting with Xiaofei, on what our procedure on breaking changes to main.

**Comment by XiaofeiCao** [comment]
I'll take this month's Compute release. We can discuss sync on the breaking change procedure on tomorrow's meeting.

**Pull request closed without merging**